### DISCUSSION TITLE
请问是否有普通用户登录自己的waline账号就能看到自己收到的评论这个功能 || Is there a function for ordinary users to log in to their waline account and see the comments they have received?

### DISCUSSION THREAD

**Discussion by changelzj** [walinejs/waline#2718] [General]
请问是否有普通用户登录自己的waline账号就能看到自己收到的评论这个功能，有时集成推送比较麻烦，用户一个一个的去文章里面主动找收到的评论也肯定不如登陆后一目了然看到自己在哪篇文章收到了哪些评论更加快捷

---
I would like to ask if there is a function for ordinary users to log in to their waline account and see the comments they have received. Sometimes it is troublesome to integrate push. It is definitely not as good as logging in to see the comments at a glance. Which article has received which comments? It is faster

**Comment by lizheming** [comment]
暂时还没有这个功能，后面看下，感觉可以修改后台的评论列表的能力，感兴趣的话也可以提 PR

---
There is no such function yet. I will look at it later and I think I can modify the comment list in the background. If you are interested, you can also submit a PR

**Reply by changelzj** [reply]
谢谢关注这个问题，期待后续更新，只是觉得普通用户是应该有个地方来查看自己的评论被谁评论过的，这样体验感更好~

---
Thank you for paying attention to this issue, and I look forward to subsequent updates. I just think that ordinary users should have a place to check who has commented on their comments, so that the experience will be better~
